docs/testing.md

```diff
@@ -0,0 +1,75 @@
+# Integration testing
+
+To make testing integrations easy, we expose some utilties from
+`@jupiterone/integration-sdk/testing` to help with performing common actions.
+
+## Utilities
+
+### Unit testing
+
+#### `createMockExecutionContext`
+
+For simple unit testing of the `getStepStartStates` and the `validateInvocation`
+functions, we expose a utility for creating a mock execution context.
+
+This function accepts options to populate the integration instance's `config`
+field to help with testing different code paths.
+
+Usage:
+
+```typescript
+// creates a simple execution context with a mocked
+// logger and instance without a configuration
+const simpleContext = createMockExecutionContext();
+/**
+ * Returns:
+ *
+ * {
+ *   logger: <mockLogger>,
+ *   instance: <local integration instance>
+ * }
+ */
```

Comment:
Nothing special should be going on here. tbh though, my editor doesn't do any sort of hovering to view docs (I use neovim). 

The only reason why I used `/** */` style comments was so that I could more easily type out the json like structure in my editor.

Since theses are in `.md` files, I don't think it should get picked up and cause any sort of strange hover doc issues.

Reply:
I wondered because there is meaning in the placement and syntax of comments, in documentation generators, which some editors understand and enhance the development experience. I thought I might not have known about a feature 😄Thanks for clarifying!